analysis/analysis_test.go

```diff
@@ -17,10 +18,19 @@ import (
 	"github.com/konveyor/tackle2-hub/api"
 	"github.com/konveyor/tackle2-hub/binding"
 	"github.com/konveyor/tackle2-hub/test/assert"
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
 )
 
 // Test application analysis
 func TestApplicationAnalysis(t *testing.T) {
+
+	// Create a kubernetes client from the default kubeconfig
+	cl, err := client.New(config.GetConfigOrDie(), client.Options{})
+	if err != nil {
+		t.Fatalf("failed to create client: %v", err)
+	}
```

Comment:
I'd suggest make this optional (use an env variable like KUBEDEBUG similar to https://github.com/konveyor/go-konveyor-tests?tab=readme-ov-file#test-execution-options, or just skip if client creation failed).

I'm sure it brings value for debugging, but not each test execution (e.g. local or jenkins) has available KUBECONFIG.

Reply:
That is good point, will make the change!